cmd/registry-replacer/main.go

```diff
@@ -64,7 +69,10 @@ func gatherOptions() (*options, error) {
 	flag.StringVar(&o.ocpBuildDataRepoDir, "ocp-build-data-repo-dir", "../ocp-build-data", "The directory in which the ocp-build-data repository is")
 	flag.StringVar(&o.currentRelease.Minor, "current-release-minor", "6", "The minor version of the current release that is getting forwarded to from the master branch")
 	flag.BoolVar(&o.pruneUnusedReplacements, "prune-unused-replacements", false, "If replacements that match nothing should get pruned from the config")
+	flag.BoolVar(&o.pruneUnusedBaseImages, "prune-unused-base-images", false, "If base images that match nothing should get pruned from the config")
+	flag.BoolVar(&o.skipApplyReplacements, "skip-apply-replacements", false, "If we should skip applying Dockerfile image replacements. You will probably not use this, and it's mostly used by tests that validate that base image pruning doesn't botch things.")
```

Comment:
All other flags are positive: "perform this step".  This one looks out of place.

We should also mention in `prune-unused-replacements` that it will not be performed if replacements are disabled.

Reply:
Yeah, I had trouble coming up with a good name for this flag. I can't really think of a good "positive" name for it. Definitely open to suggestions :)

Reply:
`--apply-replacements=false --prune-unused-base-images`?

I'm not the greatest namer of things either, but that sounds better.

Reply:
Sure, I can change it to that.